### DISCUSSION TITLE
Use "Relational Database" instead of "Database"

### DISCUSSION THREAD

**Discussion by miiitch** [microsoft/NubesGen#82] [Request for Comments]
Right now, the only databases in the list are relational databases. This would be consistant with the fact that there are other databases in the Addon list. 

What do you think?

**Comment by jdubois** [comment]
Yes, I agree we should change the wording to avoid confusion

**Comment by pmsousa** [comment]
You would consider an area just for Relational Databases and other types as Add-Ons? Or should the other types - i.e. MongoDB - be included not as an Add-On but on the Database section?
When you go to the Portal, if you select "All Services - Databases" you have a mix. For coherence, shouldn't we go for the same approach?

**Reply by jdubois** [reply]
The idea was to highlight relational databases, as those are probably what people will use, and also that normally you'd only use one relational database at a time. Then you can add other services, like MongoDB.
Of course it won't work for all scenarios, we're just trying to have something easy for the most common one.
Maybe we would need to group some add-ons by type? Or put forward the ones which are the most common?

**Reply by LayZeeDK** [reply]
I was also confused by this. It's not very clear how to choose only Azure Cosmos DB.
